cmd/apiserver-boot/boot/glide.go

```diff
@@ -63,39 +63,31 @@ func fetchGlide() {
 }
 
 func copyGlide() {
-	// copy the files
+	// Move up two directories from the location of the `apiserver-boot`
+	// executable to find the `vendor` directory we package with our
+	// releases. TODO(campbellalex@google.com): this doesn't work for people
```

Comment:
Sure, that sounds good, but I don't think it solves the issue that I'm referring to in this comment.

When you run `go install` on apiserver-builder (as a developer), you get a binary in your `GOPATH/bin` directory that does not work because it cannot find its vendor directory. Is the intention to make developers add apiserver-builder to their PATH?